### PR TITLE
Added Support for JSONArrays

### PR DESCRIPTION
**Pull request by dmuecke**
This change allows to parse arrays when using JSONParser.

### CODE REVIEW

File: gwt-test-utils/src/main/java/com/googlecode/gwt/test/internal/patchers/JSONParserPatcher.java [hunk 1]
```diff
@@ -32,9 +32,16 @@ static JSONValue evaluate(String json, boolean strict) {
                 f.configure(JsonParser.Feature.ALLOW_NUMERIC_LEADING_ZEROS, true);
             }
             jp = f.createJsonParser(json);
-            jp.nextToken(); // will return JsonToken.START_OBJECT (verify?)
-            JSONObject jsonObject = extractJSONObject(json, jp);
-            return jsonObject;
+            JsonToken token = jp.nextToken(); // will return JsonToken.START_OBJECT (verify?)
```
Comment (davidrusu) [inline gwt-test-utils/src/main/java/com/googlecode/gwt/test/internal/patchers/JSONParserPatcher.java]: Do you still need the comment here?

Comment (dmuecke) [inline gwt-test-utils/src/main/java/com/googlecode/gwt/test/internal/patchers/JSONParserPatcher.java]: You're right. I'm going to fix it.

**Comment by Gael** [comment]
Thank you for this PR !

**Comment by Gael** [comment]
I just released the 0.53 version (for GWT 2.8) with your PR. It is expected to be available in a couple of hours on Maven Central.

**Pull request merged**